api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -269,6 +269,7 @@ def create_book_in_multiple_venues(venues: list[offerers_models.Venue]) -> None:
     for venue in venues[:3]:
         offer = offers_factories.OfferFactory(
             product=product,
+            subcategoryId=subcategories_v2.LIVRE_PAPIER.id,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
```suggestion
            subcategoryId=product.subcategoryId